src/composables/use-lieux.ts

```diff
@@ -1,38 +1,34 @@
 import { ref, computed } from "vue"
-import {
-  getBenefitEtablissements,
-  getEtablissements,
-} from "@lib/benefits/etablissements"
+import { getBenefitLieuxTypes, getLieux } from "@lib/benefits/lieux"
 import { useStore } from "@/stores/index.js"
 import { useRoute } from "vue-router"
 import Individu from "@lib/individu.js"
 import { ActiviteType } from "@lib/enums/activite.js"
-import { HelpingInstitution } from "@lib/types/helping-institution.d.js"
+import { LieuLayout } from "@lib/types/lieu.d.js"
 import * as Sentry from "@sentry/vue"
 
-export function useEtablissements() {
+export function useLieux() {
   const store = useStore()
   const $route = useRoute()
 
-  const etablissements = ref<HelpingInstitution[]>([])
+  const lieux = ref<LieuLayout[]>([])
   const benefit = ref<any>(null)
   const updating = ref<boolean>(true)
 
   const city = store.situation.menage.depcom
 
-  const currentEtablissement = computed(() => {
-    return etablissements.value.find(
-      (etablissement: HelpingInstitution) =>
-        etablissement.id === $route.params.etablissement_id
+  const currentLieu = computed(() => {
+    return lieux.value.find(
+      (lieu: LieuLayout) => lieu.id === $route.params.lieu_id
     )
   })
 
-  interface EtablissementTypeCriteria {
+  interface LieuTypeCriteria {
     isRelevant?: (demandeur: any, situation: any) => boolean
     types: string[]
   }
 
-  const etablissementsList: EtablissementTypeCriteria[] = [
+  const lieuxList: LieuTypeCriteria[] = [
```

Comment:
```suggestion
  const lieuTypeCriteriaList: LieuTypeCriterion[] = [
```

Pas obligatoire, mais ça améliore encore un peu le naming je trouve. Dis moi ce que tu en pense. Le suffixe `List` devient même questionable.

Reply:
Je suis d'accord, on peut retirer le mot "list", la variable étant nommée au pluriel :)